test/unicode2latex.jl

```diff
@@ -12,3 +12,27 @@ raw"\begin{equation}
 ", "\r\n"=>"\n")
 
 @test latexify("αaβ") == raw"${\alpha}a\beta$"
+
+@test latexify("αaβ").s == raw"${\alpha}a\beta$"
+
+@test latexify("ÀéÜ"; parse=false).s == raw"$\textrm{\`{A}}\textrm{\'{e}}\textrm{\\\"{U}}$"
```

Comment:
Is there a reason for the extra `\\` with diaereses?

Reply:
Yes, you have to correctly escape `"` which is embedded in a string.